8.8/vips.modules

```diff
@@ -770,60 +651,44 @@
 
     -->
 
-  <!-- glib 2.55.0 is available but the g_stat refactoring is slightly buggy
-  https://github.com/GNOME/glib/commit/53bd6a359f2c48e7729f89902097c892c8aa6fea
```

Comment:
I'm unsure if the `g_stat` is still buggy, perhaps it has been fixed with:
https://github.com/GNOME/glib/commit/d3d6ef60c686a21de07fa4668451e8498527a776

/cc @lovell

Reply:
I hope so; the sharp unit tests will fail if not so there's one way to find out ;)

Reply:
I downloaded the [`libvips-8.7.4-win32-x64.tar.gz`](https://github.com/lovell/sharp-libvips/releases/download/v8.7.4/libvips-8.7.4-win32-x64.tar.gz) tarball and replaced all `lib/*.{dll,lib}` and `include/*` files with the 8.8.0-rc1 (`-web`) binaries from this PR. Then I re-packed it and [uploaded it to S3](https://libvips-packaging.s3.amazonaws.com/libvips-8.7.4-win32-x64.tar.gz).

The following commands:
```bash
# Note: remove C:\Users\[user]\AppData\Roaming\npm-cache\_libvips before running these commands.
git clone https://github.com/lovell/sharp.git
cd sharp
npm config set sharp_dist_base_url "https://libvips-packaging.s3.amazonaws.com/"
npm install --build-from-source
npm test
```
Outputs:
```
...

info sharp Downloading https://libvips-packaging.s3.amazonaws.com/libvips-8.7.4-win32-x64.tar.gz
...

796 passing (48s)
```
Hooray! :tada:

If you want to test this as well, you can download the binaries of this PR here:
https://libvips-packaging.s3.amazonaws.com/vips-dev-w64-web-8.8.0.zip
https://libvips-packaging.s3.amazonaws.com/vips-dev-w64-all-8.8.0.zip